styles/pov.h

```diff
@@ -4,14 +4,41 @@
 // TODO(hubbe): Make it work with FastLED
 #if defined(ENABLE_WS2811)
 
-// Usage: &style_pov or StylePtr<StylePtr>
-// return value: suitable for preset array
-
-// This style draws images in the air if you swing the saber
-// carefully back and forth. Unfortunately you can't really see
-// the result without a slow exposure camera. It's possible that
-// dotstar APA102 pixels would be fast enough to do the effect
-// full justice, but I haven't tried that.
+/* 
+This style draws images in the air if you swing the saber
+carefully back and forth. Unfortunately you can't really see
+the result without a slow exposure camera. It's possible that
+dotstar APA102 pixels would be fast enough to do the effect
+full justice, but I haven't tried that.
+
+Usage: &style_pov
+or: StylePtr<StylePOV<>>()
+or: StylePtr<ContinuousPOV<>>()
+return value: suitable for preset array
+
+StylePOV<> draws one instance of the image in the top portion of the swing "window".
+StylePOV takes four (optional) arguments:
+template<int MIN_DEGREES=-45, int MAX_DEGREES=45, bool REPEAT=false, bool MIRROR=true>
```

Comment:
This is a weird way to write it.
I would write it something like: (hopefully github doesn't eat the less-than and greater-then)

or: StylePOV<MIN_DEGREES, MAX_DEGREES, REPEAT, MIRROR>>()

MIN_DEGREES: integer (default -45)
MAX_DEGREES: integer (default 45)
REPEAT: bool (default false)
MIRROR: bool (default true)
return value: COLOR

Since MIN_DEGREES, MAX_DEGREES, REPEAT and MIRROR all have defaults, you can write it without them, like: StylePOV<>

Reply:
updated